src/runtime_src/core/edge/common_em/config.cxx

```diff
@@ -886,39 +886,37 @@ namespace xclemulation{
     return false;
   }
 
-  int getIPName2Index(const char *name,const char* buffer)
+  //Get CU index from IP_LAYOUT section for corresponding kernel name
+  int getIPName2Index(const char *name, const char* buffer)
   {
     std::string errmsg;
-    const uint64_t bad_addr = 0xffffffffffffffff;
+    const uint64_t bad_addr = -1;
   
     if (!buffer)
     {
       errmsg = "ERROR: getIPName2Index - can't load ip_layout section";
-      std::cout << errmsg << std::endl;
+      std::cerr << errmsg << std::endl;
       return -EINVAL;
     }
 
-    const ip_layout* map = (reinterpret_cast<const ::ip_layout*>(buffer));
+    auto map = (reinterpret_cast<const ::ip_layout*>(buffer));
```

Comment:
Remove outer  `()`

Reply:
removed outer() and submitted latest changes